OoyalaSkinSampleApp/app/build.gradle

```diff
@@ -54,13 +64,7 @@ task copyTask(type: Copy) {
 tasks.copyTask.execute()
 
 dependencies {
-    compile 'com.google.android.gms:play-services-ads:+'
-    compile files('libs/FWAdManager.jar')
-    compile files('libs/OoyalaFreewheelSDK.jar')
-    compile files('libs/OoyalaIMASDK.jar')
-    compile files('libs/OoyalaSDK.jar')
-    compile files('libs/ima-android-v313.jar')
-
-    compile files('libs/OoyalaSkinSDK.jar')
```

Comment:
not a big fan of this change.. why did we switch to compiling the file tree?  What happens when I want to just replace the skin jar?